iron-selection.js

```diff
@@ -84,33 +78,30 @@ IronSelection.prototype = {
         }
       }
     }
-  },
+  }
 
   /**
    * Sets the selection state for a given item. If the `multi` property
    * is true, then the selected state of `item` will be toggled; otherwise
    * the `item` will be selected.
    *
-   * @method select
    * @param {*} item The item to select.
    */
-  select: function(item) {
+  select(item) {
     if (this.multi) {
       this.toggle(item);
     } else if (this.get() !== item) {
       this.setItemSelected(this.get(), false);
       this.setItemSelected(item, true);
     }
-  },
+  }
 
   /**
    * Toggles the selection state for `item`.
    *
-   * @method toggle
    * @param {*} item The item to toggle.
    */
-  toggle: function(item) {
+  toggle(item) {
     this.setItemSelected(item, !this.isSelected(item));
   }
-
-};
+}
```

Comment:
This still needs a `;` since it's an export statement if we're trying to avoid ASI.

Reply:
Done.